quipucords/quipucords/settings.py

```
@@ -149,6 +150,9 @@ def create_random_key():
 
 # Database Management System could be 'sqlite' or 'postgresql'
 QPC_DBMS = env.str("QPC_DBMS", "postgres").lower()
+allowed_db_engines = ["sqlite", "postgres"]
+if QPC_DBMS not in allowed_db_engines:
+    raise ImproperlyConfigured(f"QPC_DBMS must be one of {allowed_db_engines}")
```

YES YES YES

FAIL FAST :clap: